block-node/stream-publisher/src/main/java/org/hiero/block/node/stream/publisher/LiveStreamPublisherManager.java

```diff
@@ -566,13 +566,15 @@ public void handleVerification(@NonNull final VerificationNotification notificat
     public void handlePersisted(@NonNull final PersistedNotification notification) {
         if (notification != null) {
```

Comment:
Move the if succeeded here to make this a bit simpler and clearer.
```suggestion
        if (notification != null && notification.succeeded()) {
```

Reply:
We can do an if null in the else; which seems more reasonable.

Reply:
I personally find that to be error prone. It is easy to miss/neglect yet anotehr null check. I think we should stick with a single point to check if the notification is null and afterwards we are safe to use the reference.

Reply:
Fair point.